### ISSUE TITLE
Will Netlify DNS support DNSSEC?

### ISSUE THREAD

**Issue by kruton** [netlify/ask-netlify#50]
## About me

- **My name:** Kenny Root
- **I've used Netlify:** lots

## My question in a little more detail

Netlify DNS has a lot of advantages such as geolocation decreasing the Time To First Byte, however it doesn't appear to support DNSSEC. Are there any plans to support it in the future?

**Comment by rootkea** [comment]
Yes, I'm also interested to know this. I was previosly using CloudFlare and now shifted to Netlify. But just now, I found out that Netlify doesn't support DNSSEC. Such a bummer!

From the last few days, since when I'm experimenting with the Netlify platform, I have come to hold the Netlify in high regard thanks to their awesome platform. Secure DNS can not be negoatibale. I expect more from you, Netlify!

**Comment by rootkea** [comment]
I have opened the feature request on Netlify Forums which is frequented by Netlify staff. Please share your views/support there: https://community.netlify.com/t/feature-requests-what-do-you-already-love-what-could-be-better/256/26
